src/components/facet/facet_group.tsx

```diff
@@ -33,17 +43,20 @@ const layoutToClassNameMap: { [layout in FacetGroupLayout]: string } = {
 export type EuiFacetGroupProps = CommonProps &
   HTMLAttributes<HTMLDivElement> & {
     layout?: FacetGroupLayout;
+    gutterSize?: EuiFacetGroupGutterSize;
```

Comment:
Always add a prop comment even if they don't exist yet in the component.

```suggestion
    /*
     * Spacing between facet items
     */
    gutterSize?: EuiFacetGroupGutterSize;
```